invenio_jsonschemas/views.py

```diff
@@ -46,9 +48,17 @@ def create_blueprint(state):
     def get_schema(schema_path):
         """Retrieve a schema."""
         try:
-            return send_from_directory(state.get_schema_dir(schema_path),
-                                       schema_path)
+            schema_dir = state.get_schema_dir(schema_path)
         except JSONSchemaNotFound:
             abort(404)
+        if request.args.get(
+            'refs',
+            current_app.config.get('JSONSCHEMAS_REPLACE_REFS'),
+            type=int
+        ):
+            schema = {'$ref': request.base_url}
```

Comment:
That was my very first approach, problem with this is that if you load the file first, then `jsonref` (even with our custom resolver) is not able to resolve references like [this one](https://github.com/inspirehep/inspire-schemas/blob/master/inspire_schemas/records/hep.json#L38).

If instead your starting point is:

```
{'$ref': 'http://localhost:5000/schemas/records/hep.json'}
```

It is able to resolve all the relative $ref (which is what we do in `Record.validate()`)

😞 🔫